spec-main/api-browser-window-spec.ts

```diff
@@ -4288,4 +4288,20 @@ describe('BrowserWindow module', () => {
       });
     });
   });
+
+  ifdescribe(process.platform === 'win32')('hookWindowMessage', () => {
+    it('can hook a window message', async () => {
+      const w = new BrowserWindow({ show: true, width: 256, height: 256 });
```

Comment:
is the window cleaned up after this run ?

Reply:
sorry, meant to mark this [wip]. i plan to add cleanup + also test `unhook`.

Reply:
got it, thanks! will dismiss my review.